canonical-websites/www.ubuntu.com
# Create the OpenStack install page

## Done
Create the OpenStack install page and removed the developer page.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/openstack/install](http://0.0.0.0:8001/openstack/install)
- Check the page matches the [copy doc](https://docs.google.com/document/d/1eQ16jzv6YRGzG_k_C-tUnhM06k6_Ic6fZM-x_ARMzuc/edit#)

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/3288
